modules/openapi-generator/src/main/resources/python/api_client.mustache

```diff
@@ -507,7 +507,7 @@ class ApiClient:
             else:
                 new_params.append((k, quote(str(v))))
 
-        return "&".join(["=".join(item) for item in new_params])
+        return "&".join(["=".join(map(str, item)) for item in new_params])
```

Comment:
Maybe we can use [`urlencode`](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlencode) instead? 

@wing328 do you know if there was a reason `urlencode` was reimplemented here?

Reply:
don't recall the reason as i wasn't the one who wrote this if i remember correctly

can urlencode handles list of values (e.g. number=1,2,3) without encoding `,` into %2C ?

Reply:
nice.

what if the input string (e.g. first element in the string array) contains ","

e.g. string=this%20is%20%2C,abc,def is the correct output as `this is ,`  is correctly encoded as `this%20is%20%2C`

agreed with you further enhancement can be done later with another PR instead

Reply:
I will need to check, hopefully there are some tests that cover all these use cases.

Reply:
i'll try to add some tests later